Sources/GRInputField/UIKit/InputFieldView.swift

```diff
@@ -501,18 +501,22 @@ internal extension InputFieldView {
     func setupCustomRightView(rightView rightSubview: UIView?) {
         /// Right view
         if let rightSubview, !isSecureTextEntry {
+            horizontalStackView.addArrangedSubview(rightView)
```

Comment:
Na riadku 347 to robi problem, ked tam rightView ostane stale? Tato funkcia moze zavolat aj viackrat (napr. pri zmene stavu vo SwiftUI), cize sa ti tam tych right view-cok popridava viacero.